tests/aftu/test_compare_graphs.py

```diff
@@ -0,0 +1,176 @@
+"""Compare graphs generated by vLLM-Spyre vs AFTU
+
+Run `python -m pytest tests/aftu/test_compare_graphs.py`.
+"""
+
+import os
+import sys
+import tempfile
+
+import pytest
+from graph_compare_utils import (collect_graph_files, compare_graphs,
+                                 get_model_path,
+                                 run_inference_py_and_get_graphs)
+from spyre_util import (generate_spyre_vllm_output, get_chicken_soup_prompts,
+                        get_spyre_model_list)
+from vllm import SamplingParams
+
+
+@pytest.mark.aftu
+@pytest.mark.parametrize("model", get_spyre_model_list())
+@pytest.mark.parametrize("backend", ["sendnn"])
```

Comment:
If this test can only run on the `sendnn` backend then we don't need to parameterize it, we can just set the backend as sendnn in the test.

Also, we should mark all tests running with the sendnn backend with `pytest.mark.spyre` for consistency with the rest of the test suite